src/index.js

```diff
@@ -0,0 +1,290 @@
+  dreamTemp.textContent = state.temp;
+};
+
+// DECREASE HUMIDITY BY 1 DEGREE
+const decreaseDreamHumidity = () => {
+  const dreamHumidity = document.getElementById("dream-humidity-percent");
+  state.humidity -= 1;
+  dreamHumidity.textContent = state.humidity;
+};
+
+// UPDATE TEMP TEXT AND CIRCLE COLOR DEPENDING ON TEMP RANGE
+const updateDreamTempColor = () => {
+  const dreamTemp = document.querySelector("#dream-temp-number");
+  const tempCircle = document.querySelector("#circle");
+
+  if (state.temp >= 80) {
+    dreamTemp.style.color = "#E0392D";
+    tempCircle.style.backgroundColor = "#E0392D";
+  } else if (70 <= state.temp && state.temp <= 79) {
+    dreamTemp.style.color = "#E0571D";
+    tempCircle.style.backgroundColor = "#E0571D";
+  } else if (60 <= state.temp && state.temp <= 69) {
+    dreamTemp.style.color = "#F09104";
+    tempCircle.style.backgroundColor = "#F09104";
+  } else if (50 <= state.temp && state.temp <= 59) {
+    dreamTemp.style.color = "#04A59F";
+    tempCircle.style.backgroundColor = "#04A59F";
+  } else if (state.temp <= 49) {
+    dreamTemp.style.color = "#385EC0";
+    tempCircle.style.backgroundColor = "#385EC0";
+  }
+};
+
+// CHANGE THE CITY NAME IN TITLE AND SEARCH BAR INPUT SIMULTANEOUSLY
+const changeCityNameWithInput = () => {
+  const cityTitle = document.getElementById("city-title");
+  const cityInput = document.getElementById("search-bar").value;
+  cityTitle.textContent = cityInput;
+};
+
+// AXIOS REQUEST
+// REQUEST LOCATION AND WEATHER FOR CITY FROM WEATHER PROXY SERVER
+const getCurrentWeather = () => {
+  let cityInput = state.city;
+
+  if (document.getElementById("search-bar").value) {
+    cityInput = document.getElementById("search-bar").value;
+  };
+
+  return axios.get('http://127.0.0.1:5000/location', {params: {q: cityInput}
+  })
+  .then(location => {
+    return {
+      lat: location.data[0].lat,
+      lon: location.data[0].lon
+    }
+  })
+  .then(coordinates => {
+    return axios.get('http://127.0.0.1:5000/weather', {
+                      params: 
+                      {lat: coordinates.lat,
+                      lon: coordinates.lon}
+                    })
+  })
+  .then(weather => {
+    return weather.data.main;
+  })
+};
+
+// STORE CURRENT WEATHER REPORT AFTER ONE CALL
+const saveCurrentWeather = () => {
+    getCurrentWeather()
+    .then(weather => {
+    state.temp = Math.floor(weather.temp);
+    state.humidity = weather.humidity;
+    updateTemp()
+    updateHumidity()
+  });
+};
+
+// AXIOS REQUEST
+// REQUEST LOCATION AND FORECAST FOR CITY FROM WEATHER PROXY SERVER
+const getFiveDayForecast = () => {
+  let cityInput = state.city;
+
+  if (document.getElementById("search-bar").value) {
+    cityInput = document.getElementById("search-bar").value;
+  };
+
+  return axios.get('http://127.0.0.1:5000/location', {params: {q: cityInput}
+  })
+  .then(location => {
+    return {
+      lat: location.data[0].lat,
+      lon: location.data[0].lon
+    }
```

Comment:
Looks good! Nice work getting this second call to work and extending your project 🥳